### PR TITLE
Arrays, Strays 1-4

### PR DESCRIPTION
**Pull request by Alexmartuhov**

### CODE REVIEW

File: src/com/Arrays/Task1.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+//Создайте массив из всех чётных чисел от 2 до 20 и выведите элементы
+//        массива на экран сначала в строку, отделяя один элемент от другого пробелом, а
+//        затем в столбик (отделяя один элемент от другого началом новой строки). Перед
+//        созданием массива подумайте, какого он будет размера.
+public class Task1 {
+    public static void main(String[] args) {
+        int number = 20;
+        int length = number / 2; //находим длину массива
+        int[] mass = new int[length];
+        mass[0] = 2;
```
Comment (svetamolchanova) [inline src/com/Arrays/Task1.java]: why you determine this value?

File: src/com/Arrays/Task3.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import java.util.Random;
+
+//Создайте массив из 15 случайных целых чисел из отрезка [0; 99]. Выведите
+//        массив на экран. Подсчитайте сколько в массиве чётных элементов и выведете
+//        это количество на экран на отдельной строке.
+public class Task3 {
+    public static void main(String[] args) {
+        Random r = new Random();
+        int[] i = new int[15];
+        int chetnie = 0;
```
Comment (svetamolchanova) [inline src/com/Arrays/Task3.java]: name value by code convension, don't use russian word with english transcription

File: src/com/Arrays/Task3.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        for (int j = 0; j < i.length; j++) {
+            i[j] = r.nextInt(99);
+            System.out.print(i[j] + " ");
+            if (i[j] % 2 == 0 && i[j] != 0) {
```
Comment (svetamolchanova) [inline src/com/Arrays/Task3.java]: it is better first check i[j] != 0, if it is false we don't need to check division

File: src/com/Arrays/Task1.java (hunk 1 above)
Comment (Alexmartuhov) [reply src/com/Arrays/Task1.java]: In the beginning i wanted to get mass[i]=mass[0]+i*2, but change my mind. 
Deleted mass[0]=2.

File: src/com/Arrays/AdditionalTask3.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import java.util.Scanner;
+
+//Найти средне арифметическое элементов массива, превосходящих некоторое
+//число С.
+public class AdditionalTask3 {
+    public static void main(String[] args) {
+        Random r = new Random();
+        Scanner sc=new Scanner(System.in);
+        System.out.println("Введите число С: ");
+        int C=sc.nextInt();
```
Comment (svetamolchanova) [inline src/com/Arrays/AdditionalTask3.java]: for variable use lowercase: int c

File: src/com/Arrays/AdditionalTask4.java [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+public class AdditionalTask4 {
+    public static void main(String[] args) {
+        Random r = new Random();
+        int[] mass = new int[20];
+        int j=0; //счетчик
+        int min=Integer.MAX_VALUE;
+        for (int i = 0; i < mass.length; i++) {
+            mass[i] = r.nextInt(100);
+            System.out.print(mass[i] + " ");
+            if(mass[i]!=0&&mass[i]%2!=0){
```
Comment (svetamolchanova) [inline src/com/Arrays/AdditionalTask4.java]: format code. Ctrl+Alt+L

File: src/com/Arrays/AdditionalTask7.java [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+import java.util.Random;
+import java.util.Scanner;
+
+//Подсчитать, сколько раз встречается элемент с заданным значением.
+public class AdditionalTask7 {
+    public static void main(String[] args) {
+        Random r = new Random();
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Введите число: ");
+        int C = sc.nextInt();
```
Comment (svetamolchanova) [inline src/com/Arrays/AdditionalTask7.java]: for variable use lowercase

File: src/com/strays/Task2.java [hunk 7]
```diff
@@ -0,0 +1,11 @@
+package com.strays;
+
+//Заменить все вхождения символа стоящего в позиции (3) на сивол стоящий в
+//        позиции 0
+public class Task2 {
+    public static void main(String[] args) {
+        String str = new String("It was the best of times, it was the worst of times,\n it was the age of wisdom, it was the age of foolishness, it was the epoch of belief");
+        String strnew = str.replace(str.substring(3, 4), str.substring(0, 1));
```
Comment (svetamolchanova) [inline src/com/strays/Task2.java]: why do you not use str.indexOf()?

File: src/com/strays/Task3.java [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+        System.out.print("Второе слово:");
+        String s2 = sc.next();
+        System.out.print("Третье слово:");
+        String s3 = sc.next();
+        String str1, str2, str3;
+        str1 = "";
+        str2 = "";
+        str3 = "";
+        for (int i = s1.length() - 1; i >= 0; i--) {
+            str1 = str1 + s1.charAt(i);
```
Comment (svetamolchanova) [inline src/com/strays/Task3.java]: you can use "+="

**Review by svetamolchanova — APPROVED** [review]

**Pull request merged**